bin/zopen-init

```diff
@@ -222,10 +214,63 @@ init()
       fi
     fi
   fi
+}
+
+determineStatsCollection()
+{
+  if isIBMHostname; then
+    isHostIBM=true
+  fi
+  if [ -n "$enableStats" ] && $enableStats; then
+    isCollectingStats=true
+  elif [ -n "$enableStats" ] && ! $enableStats; then
+    isCollectingStats=false
+  elif $isHostIBM; then
+    printAttention "Attention: Turning on usage statistics as your host is an IBM host. If this is incorrect, rerun with --noenable-stats"
+    isCollectingStats=true
+  else
+    printInfo "Do you want to contribute anonymous usage statistics to help improve z/OS Open Tools? (press y to collect stats or n for no): "
+    read collectStats < /dev/tty
+    if [ "y" = "${collectStats}" ]; then
```

Comment:
check for capital Y too and case-insensitive 'yes'

Reply:
ok, I will turn this into a function